imap/imap_private.h

```diff
@@ -234,9 +234,9 @@ struct ImapAccountData
  */
 struct ImapMboxData
 {
-  char *name;
-  char *munge_name;
-  char *real_name;
+  char *name;        /* Mailbox name */
+  char *munge_name;  /* Munged version of the mailbox name */
+  char *real_name;   /* Original Mailbox name, e.g.: INBOX can be just \0 */
```

Comment:
Please can you change the format of these comments to: `/**< desc */`
Doxygen will understand that: 'I am the docs for the thing before me'

Reply:
done